ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/result/api/PerimeterResult.java

```diff
@@ -7,30 +7,10 @@
 
 package com.farao_community.farao.search_tree_rao.result.api;
 
-import com.farao_community.farao.data.crac_api.range_action.PstRangeAction;
-import com.farao_community.farao.data.crac_api.range_action.RangeAction;
-
-import java.util.Set;
-import java.util.stream.Collectors;
-
 /**
  * This interface gives a perimeter result. It represents a unique state optimization with full network and range actions.
  *
  * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
  */
 public interface PerimeterResult extends OptimizationResult {
```

Comment:
is this interface still necessary ?

Reply:
I believe it might be used so we know where the object came from (from a whole PerimeterOptimization rather than just a Leaf for instance)